test/spec-history.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+    <script src="../../web-component-tester/browser.js"></script>
+
+    <link rel="import" href="../../polymer/polymer.html">
+    <link rel="import" href="../palindrom-redirect.html">
+</head>
+
+<body>
+    <test-fixture id="element">
+        <template>
+            <dom-bind>
```

Comment:
Is this `dom-bind` needed here for some reason? or are you just trying to recreate Starcounter's case? If the latter, I'd create two cases:
1. No `dom-bind` setting URL via attribute.
2. `dom-bind` and setting via `domBind.set('prop','/new/index.html') and `url="{{prop}}"`

Or just remove `dom-bind` and test as you do via prop and attr directly.

Reply:
There is no need. It is a mistake. In my head I wanted to use it to change the value then used the id and forgot about it 😄